# contact message describing each agent
ID robot_id
geometry_msgs/PoseWithCovariance pose
geometry_msgs/TwistWithCovariance twist
